crates/polars-plan/src/dsl/function_expr/mod.rs

```diff
@@ -18,6 +18,7 @@ mod cum;
 #[cfg(feature = "temporal")]
 mod datetime;
 mod dispatch;
+mod ewm;
```

Comment:
why not just feature flag the module instead of feature flagging the individual functions in `ewm.rs`?

Reply:
You are right, will fix